packages/bisect/bisect.1.3/opam

```diff
@@ -4,7 +4,10 @@ authors: ["Xavier Clerc"]
 homepage: "http://bisect.x9c.fr/"
 license: "GPL v3"
 build: [
-  ["sh" "configure" "-ocaml-prefix" prefix "-ocamlfind" "%{bin}%/ocamlfind"]
+  ["sh" "configure" "-ocaml-prefix" prefix "-ocamlfind" "%{bin}%/ocamlfind"] 
+    {ocaml-version != "4.00.1"}
+  ["sh" "configure" "-ocaml-prefix" prefix "-ocamlfind" "%{bin}%/ocamlfind" "-ppx"]
+    {ocaml-version = "4.00.1"}
```

Comment:
Don't quite understand what's going on here. Why only specifically pass the `-ppx` flag for `4.00.1`, and not for earlier or later versions?